code/open_file.py

```diff
@@ -0,0 +1,42 @@
+# Actions for opening files in appropriate applications.
+from talon import Module, Context, actions, ui, app
+import os
+
+# path to knausj root directory
+REPO_DIR = os.path.dirname(os.path.dirname(__file__))
+SETTINGS_DIR = os.path.join(REPO_DIR, 'settings')
+
+mod = Module()
+ctx = Context()
+
+mod.list('talon_settings_csv', desc="Absolute paths to talon user settings csv files.")
+_csvs = {
+    x: os.path.join(SETTINGS_DIR, '_'.join(x.split()) + ".csv")
+    for x in ["file extensions", "search engines", "system paths",
+              "websites", "words to replace", "additional words"]
+}
+_csvs.update({
+    "homophones": os.path.join(REPO_DIR, 'code', 'homophones.csv'),
+})
```

Comment:
I think maybe this stuff would go into a separate file, along with the proposed `edit_user_file`

Reply:
Yeah, it might belong in user_settings.py